cmd/juju/commands/main.go

```diff
@@ -142,6 +143,17 @@ func (m main) Run(args []string) int {
 		}
 	}
 
+	// See if we need to invoke the juju interactive shell.
+	// It is invoked by:
+	// $ juju
+	// $ juju --help | -h | help
```

Comment:
I don't think we should be starting the repl shell if they ask for help.

Reply:
Ah, the comment was wrong. juju --help does not invoke the repl. That was a typo on my part